src/native/crt.c

```diff
@@ -265,6 +265,15 @@ bool aws_jni_check_and_clear_exception(JNIEnv *env) {
     return exception_pending;
 }
 
+jthrowable aws_jni_get_and_clear_exception(JNIEnv *env) {
+    jthrowable exception = (*env)->ExceptionOccurred(env);
+    if (exception) {
+        (*env)->ExceptionDescribe(env);
```

Comment:
Probably don't want to call `ExceptionDescribe()` which prints things to `stderr`. It made sense for `aws_jni_check_and_clear_exception()` to do this, since we're throwing out the exception there, and there's no other way to see that anything weird happened.

But presumably with this new function we're going to actually DO something with the exception